module-ooto-ci.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Indeed, I think this file should be removed.

Reply:
Exactly, I thought the same. I think it's something that I carried from the `rebase` because I don't remember modifying this file :thinking: 
Should I remove it?

Reply:
Yes please.